### ISSUE TITLE
finding_donors features scaling

### ISSUE THREAD

**Issue by StefanoCappellini** [udacity/machine-learning#215]
I think there is an error in the "Normalizing Numerical Features" section of the finding_donors notebook. There the MinMaxScaler is applied to *data* but I think it should be better applied to features_raw, otherwise the log-transform made before would be useless.

    scaler = MinMaxScaler()  
    numerical = ['age', 'education-num', 'capital-gain', 'capital-loss', 'hours-per-week']
    features_raw[numerical] = scaler.fit_transform(data[numerical])

**Comment by pehpa** [comment]
You are right. I also created an issue concerning this problem here https://github.com/udacity/mlnd_issues_tracker/issues/9

**Comment by adarsh0806** [comment]
Code updated.
